App/Services/CameraRoll.ts

```diff
@@ -71,3 +82,34 @@ export async function chooseProfilePhoto(): Promise<{ uri: string, data: string}
     })
   })
 }
+
+export async function choosePhoto(): Promise<PickerImage> {
```

Comment:
Ok this is fine for now, but we should consolidate these two functions into one, they're basically the same.